great_expectations/cli/datasource.py

```diff
@@ -110,7 +115,7 @@ def _add_pandas_datasource_with_in_memory_generator(context):
 
     configuration = PandasDatasource.build_configuration(generators={
                                                              "default": {
-                                                                 "class_name": "InMemoryGenerator",
+                                                                 "class_name": "PassthroughGenerator",
```

Comment:
Not `ManualGenerator`?

Reply:
Based on our conversation, we are OK with PassthroughGenerator